src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -5182,42 +5215,48 @@ public void decommission(boolean force) throws InterruptedException
             }
 
             startLeaving();
-            long timeout = Math.max(RING_DELAY_MILLIS, BatchlogManager.instance.getBatchlogTimeout());
+            long timeout = Math.max(RING_DELAY_MILLIS, BatchlogManager.getBatchlogTimeout());
```

Comment:
Is there some nuance to this that I don't see, or is it unnecessary?

Reply:
@driftx IDEA was telling me that we are accessing static variable `BATCHLOG_REPLAY_TIMEOUT` via instance reference which just does not make sense, does it? It is unnecessary to do it like that.

Reply:
I see. I'm fine with keeping it, was just curious if I was missing something, thanks!